package.json

```diff
@@ -79,6 +80,8 @@
     "@types/node": "^16.10.3",
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
+    "chai": "^4.3.4",
```

Comment:
**question(non-blocking):** why adding chai ?

Where is it used in this PR ?
What is the overall value for the plugin in the future ?

Reply:
When I was initially setting up the environment to run the tests, I think something was confused on my end and it appeared to be trying to run `chai` but not able to find it. My apologies. I've removed it considering we're not using it anywhere in this project!